update_versions.py

```diff
@@ -0,0 +1,5 @@
+import json
+
+from setup import max_version, min_version, supported_python_versions
+
+print(json.dumps(supported_python_versions))
```

Comment:
This file will be used later as well to update the README.rst instructions with updated supported Python versions.